Dockerfile

```diff
@@ -24,19 +31,30 @@ RUN \
         python3 \
         python3-pip \
         python3-setuptools \
-        # git \
     && pip3 install -U pip setuptools packaging \
     && pip3 install -U psycopg2-binary \
     && pip3 install -U migra[pg] \
-    && apt-get remove -y apt-transport-https software-properties-common build-essential python3-pip python3-dev python3-setuptools \
+    && pip3 install poetry==1.1.7 \
+    && apt-get remove -y apt-transport-https \
     && apt-get -y autoremove \
     && rm -rf /var/lib/apt/lists/*
 
 EXPOSE 5432
 
 RUN mkdir -p /docker-entrypoint-initdb.d
-# COPY ./docker/initpgstac.sh /docker-entrypoint-initdb.d/initpgstac.sh
-# COPY ./pgstac.sql /workspaces/pgstac.sql
 COPY ./sql /docker-entrypoint-initdb.d/
 
-WORKDIR /workspaces
+RUN mkdir -p /opt/src/pypgstac
+
+WORKDIR /opt/src/pypgstac
+
+COPY pypgstac/poetry.lock pypgstac/pyproject.toml ./
+RUN poetry install
+
+
+COPY pypgstac /opt/src/pypgstac
+RUN poetry install
```

Comment:
Are both `poetry install` commands needed here? Wouldn't the second take care of the first since it copies the entire directory?